internal/database/migrations/migrations.go

```diff
@@ -0,0 +1,46 @@
+package migrations
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/uwblueprint/shoe-project/internal/database/models"
+	"gorm.io/gorm"
+)
+
+func CreateTables(db *gorm.DB) error {
+	return db.AutoMigrate(&models.Story{})
```

Comment:
Don't need to do the same for author table?

Reply:
weirdly works without it (briefly looked at `AutoMigrate` implementation, but nothing about foreign keys jumped out), but good catch